content/desktop/settings/mac.md

```diff
@@ -64,6 +64,10 @@ If you choose the integrated terminal, you can run commands in a running contain
 
 - **Show CLI hints**. Displays CLI hints and tips when running Docker commands in the CLI. This is turned on by default. To turn CLI hints on or off from the CLI, set `DOCKER_CLI_HINTS` to `true` or `false` respectively.
 
+- **SBOM Indexing**. When this option is enabled, inspecting an image in Docker Desktop shows a **Start analysis** button that, when selected, analyzes the image with Docker Scout.
```

Comment:
Is this just for Mac or will it be windows and linux as well?

Reply:
Oh right :D